packages/api-explorer/e2e/diffScene.spec.ts

```diff
@@ -151,7 +151,7 @@ describe('Diff Scene', () => {
           page.evaluate((el) => el.innerText.match(/^[a-z_]*/)[0], resultCard)
         )
       )
-      expect(page1Methods).toHaveLength(15)
+      expect(page1Methods).toHaveLength(16)
```

Comment:
This was failing in main, fixed so that it is up to date and accurately reflecting new additional method.